CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 **SDK**
 
 - **Feature:** Added support for Orientation lock, that enables the mini app to lock `portrait` or `landscape` orientation for the mini-app. [Please check here](USERGUIDE.md#orientation-lock)
+- **Feature:** Added default implementation in SDK for `requestCustomPermissions(permissions:miniAppTitle:completionHandler:)` [Please check here](USERGUIDE.md#request-custom-permission)
+- **Deprecated:** `requestCustomPermissions(permissions:completionHandler:)` in `MiniAppCallbackProtocol` protocol is deprecated
```

Comment:
One qus, do we need to mark the interface as deprecated? In my understanding, HostApp still can use it when they want to implement their own UI.

Reply:
I think that this is deprecated because it's been replaced by a new method - so it is still possible for the host app to override it.

@leorjoseph Maybe add something like this here: `You should use requestCustomPermissions(permissions: miniAppTitle:completionHandler:) instead`

Reply:
@corycaywood It is already added to the developer documentation. Sure, I will add it here as well 👍